superset-frontend/src/pages/Tags/index.tsx

```diff
@@ -43,6 +43,21 @@ import { Tag as AntdTag } from 'antd';
 import { Tag } from 'src/views/CRUD/types';
 import TagCard from 'src/features/tags/TagCard';
 
+const emptyState = {
+  title: t('No Tags created'),
+  image: 'dashboard.svg',
```

Comment:
Do we have a different image to use here?

Reply:
yes but this is what @kasiazjc wanted for the empty for tags